FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Ayaz Rahman on 16/7/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct FeedImageMapper {
```

Comment:
`FeedImageMapper` is defined as a struct, so we can instantiate it as `FeedImageMapper()`. However, all its methods are static - it has no instance methods or properties.

It may be confusing for other developers to be able to instantiate a struct but then not be able to do anything with it.

As you'll see in the following lectures, you can make the initializer private in such cases since the `FeedImageMapper` is just a namespace for static methods, with no instance-specific behavior.

You can use a private initialize in a struct or a class, or even make it an enum with no cases, which can never be instantiated!

These solutions will disallow instantiation at compile time so that `FeedImageMapper()` won't compile ✅

Reply:
The convention in this codebase is to omit the default access leve keyword `internal`, please remove it.
```suggestion
struct FeedImageMapper {
```